### PR TITLE
[Backport 2.4] Add window and mac platform to run unit test (#74)

### PR DESCRIPTION
**Pull request by heemin32**
Add window and mac platform to run unit test

Signed-off-by: Heemin Kim <heemin@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by vamshin — APPROVED** [review]
LGTM! Thanks

File: src/plugins/custom_import_map/test/jest.config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
-
+process.env.TZ = 'UTC';
```
Comment (martin-gaievski) [inline src/plugins/custom_import_map/test/jest.config.js:5]: want to confirm that this file is executed by some other process and all vars are set and this is exact equivalent of previous syntax TZ=UTC

Comment (heemin32) [reply src/plugins/custom_import_map/test/jest.config.js:5]: This apply only for test. Don't have full understanding of how this value is being used in other places tbh but at least all tests are passed.

Comment (heemin32) [reply src/plugins/custom_import_map/test/jest.config.js:5]: This PR is already merged in main and 2.x branch. If there is any doubt, we can create a new issue and work on a new thread.

**Review by martin-gaievski — APPROVED** [review]

**Pull request merged**